Automattic/pocket-casts-android
# Onboarding: Update login screen artwork

## Description

Updates login artwork.

## Testing Instructions

1. Fresh install and open the app
2. Notice that the artwork is displayed on the initial onboarding sign in / sign up screen
3. Notice that artwork is adjusted based on whether Google sign in button is shown or not on the screen

## Screenshots or Screencast 

Phone (Nexus 5)

with_google_sign_in_button| without_google_sign_in_button
----|-----
<img height=420 src="https://user-images.githubusercontent.com/1405144/207871723-397adae4-1af9-439d-b8d1-a8953e82a3b8.png"/>| <img height=420 src="https://user-images.githubusercontent.com/1405144/207871796-f794bb58-8c5d-485d-b091-d653f6f251f6.png"/>

Tablet (Nexus 9)

with_google_sign_in_button| without_google_sign_in_button
----|-----
![with_google_tablet](https://user-images.githubusercontent.com/1405144/207873367-ee09b610-a5e8-4677-962e-1ae9d042cabd.png)| ![without_google_tablet](https://user-images.githubusercontent.com/1405144/207873412-cfbe50cd-b06a-4263-9e30-67d099fe8a6c.png)
